language.txt

```diff
@@ -76,6 +76,31 @@ Unicode code point."
 `AbCdEf`
 
 
+# Bit string literals.
+
+# A bit string literal is a hex literal beginning with 'b'. Only 0 and 1 may
+# appear inside. Bit string literals are encoded like a BIT STRING. (Big-endian
```

Comment:
| can also appear inside, at which point it's not really a hex literal anymore. I also went a bit wordy in the others (probably could have picked a tighter phrasing...) because "encoded like a BIT STRING" could mean with or without the tag. Perhaps:

A backtick string beginning with 'b' is a bit string literal. 0 or 1 characters denote bits in a bit string. | characters are interpreted as below. No other characters may appear. Bit string literals emit the contents of the bit string's DER encoding as a BIT STRING. (Big-endian bit order, zero-padded, and prefixed with the number of padding bits.)

Reply:
Much better! I clamped down the wording on the other paragraph too.